source/Outfit.cpp

```diff
@@ -334,13 +339,20 @@ bool Outfit::IsDefined() const
 
 // When writing to the player's save, the reference name is used even if this
 // outfit was not fully defined (i.e. belongs to an inactive plugin).
-const string &Outfit::Name() const
+const string &Outfit::TrueName() const
 {
 	return name;
```

Comment:
`TrueName()` now returns `name`, while `Name()` returns `displayName`...

Renaming the variable named `name` to `trueName` might help to avoid some bugs in the future.

Reply:
I'd be fine with doing this as part of the other issue.

Reply:
done